docs/overview/Introduction.md

```diff
@@ -28,9 +28,47 @@ A CentrifugeID has the following credentials:
 
 * Ethereuem Accounts: When interacting with a smart contract on Ethereum, an account needs to be linked to the identity to act on its behalf. The linked Ethereum accounts are the accounts that are allowed to interact with DApps utilizing Centrifuge OS.
 
+Centrifuge OS users register their identity on the public Ethereum network and
+maintain a set of public keys as part of their identity. These identities and their keys are used to resolve nodes on the P2P layer when one node wants to transmit data to
+another Centrifuge user. This means that nodes on the P2P layer find others by
+looking up their public keys from the public identity register based on CentrifugeID.
+
+## Centrifuge Peer-to-Peer Network
+
+A peer-to-peer (P2P) network consists of two or more systems connected and sharing resources without going through a separate server. Centrifuge consists of a P2P network for decentralized data exchange using smart contracts on Ethereum. The Centrifuge P2P node supports handling of incoming and outgoing data using a service bus principal where plugins and outside systems can subscribe to messages about specific objects.
+
 ## Centrifuge Node
 
 The Centrifuge Node serves an interface to the upstream system in addition to being a part of the peer to peer network that forms the Centrifuge OS ecosystem.<!-- (Need more information here).-->
 
+## Centrifuge Sidechain
+
+The Centrifuge sidechain is a public Ethereum-based blockchain that deploys proof-of-authority consensus algorithm. The collaborators operating on the sidechain are voted in or out based on a token curated registry (TCR) located on the Ethereum mainnet.
+
+## Merkel Tree
+
+In a block, the Merkel tree is a tree in which every lead node is assigned a hash of the data block and every non-leaf node is labelled with a cryptographic hash of the lables of its child nodes. Merkel trees allow you to validate that any element in a large set of values is a part of one Merkel root hash. In the Ethereum blockchain, Merkle trees are created for each block to verify that a specific transaction is part of a mined block.
+
+Centrifuge OS uses Merkel trees fir proving data validation for documents. In the Centrifuge OS, Merkel trees can be used to share the subset of a document with a third party to prove authenticity of the whole data structure while keeping the rest of the document private.
```

Comment:
Centrifuge OS uses _Merkle_ trees _for_ proving data validation for documents. In the Centrifuge OS, _Merkle_ trees can be used to share the subset of a document with a third party to prove authenticity of the whole data structure while keeping the rest of the document private.

Reply:
Done